paddlespeech/t2s/exps/inference.py

```diff
@@ -112,7 +112,7 @@ def parse_args():
     parser.add_argument(
         "--device",
         default="gpu",
-        choices=["gpu", "cpu", "xpu", "npu", "mlu"],
+        choices=["gpu", "cpu", "xpu", "npu", "mlu", "gcu"],
```

Comment:
Can GCU be used locally to run?

Reply:
Yes， it runs well with an extra option '--gcu'。We have tried to run the scenarioslocally including Speech Recognition,Speech Translation (English to Chinese),Text-to-Speech and Punctuation Restoration.But there is one thing to note that it depend on paddlepaddle cpu version.Currentlly PaddleSpeech depends on paddlepaddle-gcu version default, we can't find a method to compatible with both scenarios when using pip to install paddlespeech so do not adjust the package depend question.